src/parse/matches/arg_matches.rs

```diff
@@ -988,7 +1003,8 @@ impl ArgMatches {
 #[derive(Clone)]
 #[allow(missing_debug_implementations)]
 pub struct Values<'a> {
-    iter: Map<Iter<'a, OsString>, fn(&'a OsString) -> &'a str>,
+    #[allow(clippy::type_complexity)]
+    iter: Map<Flatten<std::slice::Iter<'a, Vec<OsString>>>, for<'r> fn(&'r OsString) -> &'r str>,
```

Comment:
Why has this become so complex?

Reply:
The only thing changed is a flatten process (Introduces a Flatten and a Vec<OsString>).